src/test/resources/config/archiver/debug-mode.conf

```diff
@@ -19,9 +19,20 @@ saps_temp_storage_path=src/test/resources/archiver-test
 # Configuração do sft
 permanent_storage_debug_tasks_dir=trash
 permanent_storage_tasks_dir=archiver
-swift_container_name=saps-test
+openstack_object_store_service_container_name=saps-test
 swift_input_pseud_folder_prefix=
 swift_output_pseud_folder_prefix=
+<<<<<<< HEAD
```

Comment:
it seems you guys had a merge issue here

Reply:
Done